cmds/grpc-backend/main.go

```diff
@@ -273,6 +283,22 @@ func RunGRPCServer(ctx context.Context, ctxCanceler func(), address string, loca
 	return s.Serve(l)
 }
 
+type GarbageCollectorJob struct {
+	name string
+	gc   ridc.GarbageCollector
```

Comment:
Rename to RIDGarbageCollectorJob since it only cleans up RID stuff otherwise we would want to make this generic and take both ridc.GarbageCollector and scdc.GarbageCollector when its available